packages/plugins/inngest/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "@envelop/inngest",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```
Let's keep this `0` and create a major changeset for `@envelop/ingest` to let changeset bump this version during the release.
You can use `yarn changeset`.

---

Added changeset.